File: src/main/java/com/example/employeeWork/entity/Manager.java
package com.example.employeeWork.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "Manager")
public class Manager {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long Id;
    @Column(nullable = false)
    private String firstName;
    @Column(nullable = false)
    private String lastName;
    @Column(nullable = false)
    private String sex;
    @Column(nullable = false, unique = true)
    private String email;

//    @OneToMany
//    @JoinColumn( name = "EmployeeId")
//    private Employee employee;
}

File: src/main/java/com/example/employeeWork/entity/Manager.kt
package com.example.employeeWork.entity

import jakarta.persistence.*
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "Manager")
class Manager {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val Id: Long? = null

    @Column(nullable = false)
    private val firstName: String? = null

    @Column(nullable = false)
    private val lastName: String? = null

    @Column(nullable = false)
    private val sex: String? = null

    @Column(nullable = false, unique = true)
    private val email: String? = null
    //    @OneToMany
    //    @JoinColumn( name = "EmployeeId")
    //    private val Employee;
}
